tools/alt-testing/delete-projects.sh

```diff
@@ -0,0 +1,3 @@
+gcloud projects delete -q ${DM_PROJECT_ID}
```

Comment:
this file should be marked as executable as well, suggest running git update-index --chmod=+x `tools/alt-testing/delete-projects.sh`

Reply:
done